Gemfile

```diff
@@ -96,6 +96,8 @@ gem 'zxcvbn', '0.1.12'
 group :development do
   gem 'better_errors', '>= 2.5.1'
   gem 'derailed_benchmarks'
+  # Putting foreman here saves developers from having bin/setup when the Ruby version changes
+  gem 'foreman', require: false
```

Comment:
We _might_ want to change the `foreman` invocation in the Makefile to `bundle exec foreman` in the case the gem is installed locally. However I think that will error out if the gem is installed system-wide.

Reply:
That's a good point. In my environment, I tried adding this line to the Gemfile to see if using `bundle exec foreman` was required, and I didn't need it. I am using `rbenv`, and I'd like to see if anyone not using `rbenv` also finds this okay.

<details><summary>Here's what happened when I added it to the Gemfile before and after running `bundle install`</summary>

```
✗ make run
yarn generate-browsers-json
yarn run v1.22.22
$ ./scripts/generate-browsers-json.js
✨  Done in 0.19s.
foreman start -p 3000
rbenv: foreman: command not found

The `foreman' command exists in these Ruby versions:
  3.2.0
  3.2.2
  3.3.0
  3.3.1
  3.3.4

make: *** [run] Error 127
➜  identity-idp git:(main) ✗ bundle
Fetching gem metadata from https://rubygems.org/........
Resolving dependencies...
Fetching foreman 0.88.1
Installing foreman 0.88.1
WARN: Unresolved or ambiguous specs during Gem::Specification.reset:
      stringio (>= 0)
      Available/installed versions of this gem:
      - 3.1.2
      - 3.1.1
WARN: Clearing out unresolved specs. Try 'gem cleanup <gem>'
Please report a bug if this causes problems.
Bundle complete! 127 Gemfile dependencies, 279 gems now installed.
Gems in the groups 'deploy' and 'production' were not installed.
Use `bundle info [gemname]` to see where a bundled gem is installed.
➜  identity-idp git:(main) ✗ make run
foreman start -p 3000
13:22:49 web.1    | started with pid 5871
13:22:49 worker.1 | started with pid 5872
13:22:49 js.1     | started with pid 5873
13:22:49 css.1    | started with pid 5874
13:22:49 js.1     | yarn run v1.22.22
13:22:49 css.1    | yarn run v1.22.22
```

[etc]
</details>

Reply:
Foreman [recommends](https://github.com/ddollar/foreman?tab=readme-ov-file#installation) against putting it in the Gemfile:

> Ruby users should take care not to install foreman in their project's Gemfile. See this [wiki article](https://github.com/ddollar/foreman/wiki/Don't-Bundle-Foreman) for more details.

The reasoning described in the wiki article seems sesnsible enough to me to not include it.